assets/Yves/js/module/checkout-widgets/address-book.js

```diff
@@ -0,0 +1,41 @@
+new OffAmazonPayments.Widgets.AddressBook({
+    sellerId: amazonpayConfig.sellerId,
+    scope: 'profile payments:widget payments:shipping_address payments:billing_address',
+    language: amazonpayConfig.locale,
+    onOrderReferenceCreate: function(orderReference) {
+        // Here is where you can grab the Order Reference ID.
+        var aoid = orderReference.getAmazonOrderReferenceId();
+
+        $(function() {
```

Comment:
I'm not sure why do you need document.ready here?

Reply:
right, not needed here